sxg_rs/src/headers.rs

```diff
@@ -114,14 +114,11 @@ impl Headers {
                     return Err(anyhow!(r#"The {} header is "{}"."#, k, v));
                 }
             }
-            // TODO: Remove this section once https://crbug.com/1250532 is fixed in most clients.
+            // https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.4
+            // https://datatracker.ietf.org/doc/html/rfc7234#section-4.1
             if let Some(vary) = self.0.get("vary") {
-                if let Ok(directives) = parse_vary_header(vary) {
-                    if directives.contains(&"*")
-                        || directives.iter().any(|d| d.eq_ignore_ascii_case("cookie"))
-                    {
-                        return Err(anyhow!(r#"The response may vary by cookie."#));
-                    }
+                if vary == "*" {
```

Comment:
Pedantic but I think this should be vary.contains because `vary: a, *` is also a valid header.

Reply:
Good catch; fixed. It seems that the semantics of that are not defined by [RFC7231 section 7.1.4](https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.4), but they are defined by the [draft httpbis spec](https://httpwg.org/http-core/draft-ietf-httpbis-cache-latest.html#caching.negotiated.responses), as [tested by cache-tests.fyi](https://cache-tests.fyi/#vary-parse).